docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -2524,10 +2524,10 @@
         "type" : "object",
         "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:JobExceptionsInfo:ExecutionExceptionInfo",
         "properties" : {
-          "exception" : {
+          "endpoint" : {
             "type" : "string"
           },
-          "location" : {
```

Comment:
I see from the proposal that location will be changed to endpoint for consistency. I see there is a V1 dispatcher.
I was expecting that the existing API would be unchanged so that existing users of the API will not be effected. I was expecting the new consistent API to all be in a higher version, so that users of the new API shape can use this version.   We could then deprecate the v1 API.
I was looking for the tests to confirm that the original API performed the same and the new consistent API performed as expected.

Reply:
Yes, it should. Thanks for catching this, I'll fix it